### PR TITLE
Small fixes to the travis build.

### PR DESCRIPTION
**Pull request by mithro**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,5 +10,4 @@ notifications:
     email:
         - hdmi2usb-spam@googlegroups.com
     irc:
-        - "chat.freenode.net#timvideos"
```
Comment (shenki) [inline .travis.yml:13]: Why did we remove it from this channel? I don't think the project is big enough that we need separate channels

Comment (mithro) [inline .travis.yml:13]: I removed it because you were complaining about the spam.

Comment (mithro) [inline .travis.yml:13]: I've added it back now but use a 1 line template rather than the default 3 line one.

**Comment by mithro** [comment]
Closing this as the [HDMI2USB-jahanzeb-firmware](https://github.com/timvideos/HDMI2USB-jahanzeb-firmware) is no longer under development. 

All active development is now on the [HDMI2USB-misoc-firmware](https://github.com/timvideos/HDMI2USB-misoc-firmware) which supports both the [Atlys](https://hdmi2usb.tv/digilent-atlys/) and [Opsis](https://hdmi2usb.tv/numato-opsis/) boards and has loads of other new features!

**Pull request merged**